guides/source/configuring.md

```diff
@@ -157,6 +157,8 @@ defaults to `:debug` for all environments. The available log levels are: `:debug
 
 * `config.time_zone` sets the default time zone for the application and enables time zone awareness for Active Record.
 
+* `config.render_hints` determines whether to output debug html comments denoting the beginning/end rendering of a partial.
```

Comment:
this should be `config.action_view.render_hints`

Reply:
Corrected.